py-polars/polars/internals/io.py

```diff
@@ -23,8 +32,22 @@
     from polars.polars import parquet_schema as _parquet_schema
 
 if TYPE_CHECKING:
+    from xlsxwriter import Workbook  # type: ignore[import]
+    from xlsxwriter.worksheet import Worksheet  # type: ignore[import]
```

Comment:
Why is this import ignored? If `xlsxwriter` has no type hints, we should just add it to the `pyproject.toml` as a module that should be ignored.

Reply:
Apparently I was distracted by the `xlsx2csv` import doing the same thing - have moved both into `pyproject.toml` now ;)